src/MDFTextAccessibility.m

```diff
@@ -155,10 +155,31 @@ + (BOOL)textColor:(nonnull UIColor *)textColor
 }
 
 + (BOOL)isLargeForContrastRatios:(nullable UIFont *)font {
+  if (font.pointSize >= 18) {
+    return YES;
+  }
+  if (font.pointSize < 14) {
+    return NO;
+  }
+
   UIFontDescriptor *fontDescriptor = font.fontDescriptor;
-  BOOL isBold =
-      (fontDescriptor.symbolicTraits & UIFontDescriptorTraitBold) == UIFontDescriptorTraitBold;
-  return font.pointSize >= 18 || (isBold && font.pointSize >= 14);
+  if ((fontDescriptor.symbolicTraits & UIFontDescriptorTraitBold) == UIFontDescriptorTraitBold) {
+    return YES;
+  }
+
+  // We treat medium as large for accesibility when larger than 14.
```

Comment:
i reopened the internal b/34075210 but as it stands now, it looks to be for all fonts.